### ISSUE TITLE
json schema

### ISSUE THREAD

**Issue by tyrasd** [digidem/osm2obj#1]
I'm wondering if there is a reason why you are using your own osm-json schema? Instead of the already existing [XAPI](https://wiki.openstreetmap.org/wiki/Xappy.js#JSON_output) / [Overpass API](http://overpass-api.de/output_formats.html#json) json format?

~~edit/ps: also, the square brackets around the tags object seems quite superfluous. ;)~~

```
  "tags": [{
    "k": "name",
    "v": "York St"
  }]
```

**Comment by astro** [comment]
There can be multiple tags, eg.:

``` json
"tags": [{
    "k": "recycling:clothes",
    "v": "no"
  }, {
    "k": "recycling:glass",
    "v": "yes"
  }]
```

Other than that, I'm fully in favour of shared schemas. (Just passing by as I noted there are already implementations to what I just hacked :-)

**Comment by gmaclennan** [comment]
Fixed in v1.0.0
